bundler/lib/bundler/gem_helper.rb

```diff
@@ -102,19 +106,25 @@ def install_gem(built_gem_path = nil, local = false)
       Bundler.ui.confirm "#{name} (#{version}) installed."
     end
 
-    def build_checksum(built_gem_path = nil)
+    def build_checksums(built_gem_path = nil)
       built_gem_path ||= build_gem
-      SharedHelpers.filesystem_access(File.join(base, "checksums")) {|p| FileUtils.mkdir_p(p) }
```

Comment:
Why was this moved inside the `write_checksums` method? It should be enough to run it just once I think?

Reply:
Good point.  Oversight.